app/controllers/verify/review_controller.rb

```diff
@@ -71,7 +70,7 @@ def idv_params
 
     def phone_confirmation_required?
       idv_params[:phone] != current_user.phone &&
```

Comment:
do we need this to also be a string rather than a symbol now? (`idv_params` is defined as `idv_session.params`, which no longer has `symbolize_keys!` being called on it)